### PR TITLE
new RPC to remove access key 

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

File: test/integration.test.js [hunk 1]
```diff
@@ -245,8 +245,44 @@ describe('with access key', function () {
...
+        const contract = await nearjs.loadContract(contractId, {
+            viewMethods: ['getValue'],
+            changeMethods: ['setValue'],
+            sender: newAccountId,
+        });
+        await account.removeAccessKey(
+            newAccountId,
+            (await keyStore.getKey(newAccountId)).getPublicKey()
+        );
+        // Replacing public key for the account with the access key
```
Comment (vgrichina) [inline test/integration.test.js]: > test that load contract works and we can make calls afterwards
and
> Replacing public key for the account with the access key

should be separate tests with common setup

File: test/integration.test.js [hunk 2]
```diff
@@ -245,8 +245,44 @@ describe('with access key', function () {
...
+        await keyStore.setKey(newAccountId, keyForAccessKey);
+        try {
+            const setCallValue2 = await generateUniqueString('setCallPrefix');
+            await contract.setValue({ value: setCallValue2 });
+            fail('Transaction signed by the access key should not have been accepted');
+        } catch (_e) {
```
Comment (vgrichina) [inline test/integration.test.js]: Jest has specific way for asserting rejected promise:

`await expect(contract.triggerAssert()).rejects.toThrow(/Transaction .+ failed.+expected to fail/);`

File: test/integration.test.js (hunk 1 above)
Comment (janedegtiareva) [reply test/integration.test.js]: this is not actually testing that we can make calls. Copy paste artifact. I updated the comment.

File: test/integration.test.js (hunk 2 above)
Comment (janedegtiareva) [reply test/integration.test.js]: done

**Pull request merged**